modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java

```diff
@@ -314,12 +316,17 @@ private enum DDMExpressionFunction {
 				"that-can-be-used-with-other-validation-functions",
 			"sum");
 
-		public static List<Map<String, String>> getItems(Locale locale) {
+		public static List<Map<String, String>> getItems(
+			String excludedDDMExpressionFunctionKey, Locale locale) {
```

Comment:
Can we change this method to make it more flexible?
For example, what if we want to hide more than one function?